modules/ROOT/pages/syntax-quick-reference.adoc

```diff
@@ -0,0 +1,1290 @@
+
+.Ordered list
+[#ex-ol]
+----
+include::lists:example$ordered.adoc[tag=nest]
+----
+
+.View result of <<ex-ol>>
+[%collapsible.result]
+====
+include::lists:example$ordered.adoc[tag=nest]
+====
+
+.Ordered list max level nesting
+[#ex-ol-max]
+----
+include::lists:example$ordered.adoc[tag=max]
+----
+
+.View result of <<ex-ol-max>>
+[%collapsible.result]
+====
+include::lists:example$ordered.adoc[tag=max]
+====
+
+Ordered lists support xref:lists:ordered.adoc#styles[numeration styles] such as `lowergreek` and `decimal-leading-zero`.
+
+.Checklist
+[#ex-check]
+----
+include::lists:example$checklist.adoc[tag=check]
+----
+
+.View result of <<ex-check>>
+[%collapsible.result]
+====
+include::lists:example$checklist.adoc[tag=check]
+====
+
+.Description list
+[#ex-dlist]
+----
+include::lists:example$description.adoc[tag=qr-base]
+----
+
+.View result of <<ex-dlist>>
+[%collapsible.result]
+====
+include::lists:example$description.adoc[tag=qr-base]
+====
+
+.Question and answer list
+[#ex-qa]
+----
+include::lists:example$description.adoc[tag=qa]
+----
+
+.View result of <<ex-qa>>
+[%collapsible.result]
+====
+include::lists:example$description.adoc[tag=qa]
+====
+
+.Mixed
+[#ex-mixed]
+----
+include::lists:example$description.adoc[tag=3-mix]
+----
+
+.View result of <<ex-mixed>>
+[%collapsible.result]
+====
+include::lists:example$description.adoc[tag=3-mix]
+====
+
+TIP: Lists can be indented.
+Leading whitespace is not significant.
+
+.Complex content in outline lists
+[#ex-complex]
+----
+include::lists:example$complex.adoc[tag=b-complex]
+----
+
+.View result of <<ex-complex>>
+[%collapsible.result]
+====
+include::lists:example$complex.adoc[tag=b-complex]
+====
+
+== Links
+
+.Autolinks, URL macro, and mailto macro
+[#ex-urls]
+----
+include::macros:example$url.adoc[tag=b-base]
+
+include::macros:example$url.adoc[tag=b-scheme]
+----
+
+.View result of <<ex-urls>>
+[%collapsible.result]
+====
+include::macros:example$url.adoc[tag=b-base]
+
+include::macros:example$url.adoc[tag=b-scheme]
+====
+
+.URL macros with attributes
+[#ex-linkattrs]
+----
+include::macros:example$url.adoc[tag=b-linkattrs]
+----
+
+.View result of <<ex-linkattrs>>
+[%collapsible.result]
+====
+include::macros:example$url.adoc[tag=b-linkattrs]
+====
+
+CAUTION: If the linked text contains a comma and the text is followed by one or more named attributes, you must enclose the text in double quotes.
+Otherwise, the text will be cut off at the comma (and the remaining text will get pulled into the attribute parsing).
+
+.URLs with spaces and special characters
+----
+include::macros:example$url.adoc[tag=b-spaces]
+----
+
+.Link to relative page
+----
+link:index.html[Docs]
+----
+
+.Link using a Windows path
+----
+include::macros:example$url.adoc[tag=b-windows]
+----
+
+.Inline anchors
+----
+include::attributes:example$id.adoc[tag=anchor]
+----
+
+.Cross references
+[#ex-xrefs]
+----
+include::macros:example$xref.adoc[tag=b-base]
```

Comment:
Should we promote `xref` instead? Or at least mention `xref`?

Reply:
Yes, absolutely. This example is outdated.

Reply:
:+1: